src/tests/locationChooser.test.ts

```diff
@@ -0,0 +1,763 @@
+      await expect(
+        locationChooser.getOrganization(ORGANIZATION_NAME)
+      ).toBeHidden();
+      await expect(
+        locationChooser.getOrganization(ORGANIZATION_NAME_SECOND)
+      ).toBeVisible();
+      await locationChooser.getOrganization(ORGANIZATION_NAME_SECOND).click();
+      await expect(
+        locationChooser.getLocationGroup('No Location Group')
+      ).toBeVisible();
+      await expect(locationChooser.getLocation(LOCATION_NAME)).toBeVisible();
+      await locationChooser.closeLocationChooserButton.click();
+    });
+
+    test('Assert updated organization for location on location chooser, react page', async ({
+      page,
+      navbar,
+      locationChooser,
+    }) => {
+      await page.goto('./dashboard');
+
+      await navbar.getNavItem('Dashboard').click();
+      await navbar.locationChooserButton.click();
+      await expect(
+        locationChooser.getOrganization(ORGANIZATION_NAME)
+      ).toBeHidden();
+      await expect(
+        locationChooser.getOrganization(ORGANIZATION_NAME_SECOND)
+      ).toBeVisible();
+      await locationChooser.getOrganization(ORGANIZATION_NAME_SECOND).click();
+      await expect(
+        locationChooser.getLocationGroup('No Location Group')
+      ).toBeVisible();
+      await expect(locationChooser.getLocation(LOCATION_NAME)).toBeVisible();
+      await locationChooser.closeLocationChooserButton.click();
+    });
+
+    test('Assert created Ward on location chooser, log in', async ({
+      browser,
+    }) => {
+      const newCtx = await browser.newContext({
+        storageState: { cookies: [], origins: [] },
+      });
+      const newPage = await newCtx.newPage();
+      const loginPage = new LoginPage(newPage);
+      const locationChooser = new LocationChooser(newPage);
+
+      await test.step('Login as new created user', async () => {
+        await loginPage.goToPage();
+        await loginPage.fillLoginForm(
+          AppConfig.instance.users.main.username,
+          AppConfig.instance.users.main.password
+        );
+        await loginPage.loginButton.click();
+      });
+      await expect(
+        locationChooser.getOrganization(ORGANIZATION_NAME)
+      ).toBeHidden();
+      await expect(
+        locationChooser.getOrganization(ORGANIZATION_NAME_SECOND)
+      ).toBeVisible();
+      await locationChooser.getOrganization(ORGANIZATION_NAME_SECOND).click();
+      await expect(
+        locationChooser.getLocationGroup('No location Group')
+      ).toBeVisible();
+      await expect(locationChooser.getLocation(LOCATION_NAME)).toBeVisible();
+      await expect(locationChooser.yourLastSingInInfo).toBeVisible();
+      await expect(locationChooser.locationChooserLogoutButton).toBeVisible();
+      await newCtx.close();
+    });
+  });
+});
+
+test.describe('Check if non manage inventory location is present in location chooser', () => {
+  const LOCATION_NAME = 'E2E-test-Depot';
+  const ORGANIZATION_NAME = '1000bulbs.com';
```

Comment:
I think we should not use this existing organization and instead use the one created by the test